### PR TITLE
Add fluid media utility

### PR DESCRIPTION
**Pull request by austinstarin**

### CODE REVIEW

File: src/sass/utilities/_media.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.u-img-fluid {
+  width: 100%;
+  max-width: auto;
```
Comment (abea) [inline src/sass/utilities/_media.scss]: Any reason the max-width shouldn't be `none`? I guess it's the same, but if the explicit purpose is that this be fluid with the container than none seems appropriate.

Comment (bgantick) [reply src/sass/utilities/_media.scss]: true - i was thinking `auto` was the default, but `none` is

Comment (austinstarin) [reply src/sass/utilities/_media.scss]: See 2546a75

Comment (bgantick) [reply src/sass/utilities/_media.scss]: 🚢

**Pull request merged**